app/app.js

```diff
@@ -20,40 +20,39 @@ app.get("/api", getApi);
 
 app.get("/api/articles/:article_id", getArticleById);
 
-app.get("/api/articles", getArticles)
+app.get("/api/articles", getArticles);
 
-app.get("/api/articles/:article_id/comments", getComments)
+app.get("/api/articles/:article_id/comments", getComments);
 
-app.post("/api/articles/:article_id/comments", postComment)
+app.post("/api/articles/:article_id/comments", postComment);
 
-app.patch("/api/articles/:article_id", patchArticle)
+app.patch("/api/articles/:article_id", patchArticle);
 
-app.delete("/api/comments/:comment_id", deleteComment)
+app.delete("/api/comments/:comment_id", deleteComment);
 
-app.get("/api/users", getAllUsers)
+app.get("/api/users", getAllUsers);
 
 app.use((err, request, response, next) => {
   if (err.code === "22P02") {
     response.status(400).send({ message: "bad request" });
-  } 
-  else if (err.code === '23502'){
-    response.status(400).send({ message: "bad request" })
-  }
-  else if(err.code === '42703'){
-    response.status(400).send({ message: "bad request" })
-  }
-  else {
+  } else if (err.code === "23502") {
+    response.status(400).send({ message: "bad request" });
```

Comment:
What is the point in differentiating if you are sending the same thing back? These error messages could be specific to what has gone wrong.